src/styles-breakpoints.scss

```diff
@@ -0,0 +1,41 @@
+// setting breakpoint sizes here
+$xs: 0px;
+$sm: 480px;
+$md: 600px;
+$lg: 1024px;
+$xl: 1366px;
+
+$breakpoint-names: 'xs', 'sm', 'md', 'lg', 'xl';
+$breakpoint-sizes: $xs, $sm, $md, $lg, $xl;
+
+@mixin breakpoint($name, $mode) {
+  $size: nth($breakpoint-sizes, $index);
+  @if ($mode== 'up') {
```

Comment:
should we choose only one breakpoint between max-width or min-width since both can be used alternatively?

Reply:
ผมคิดว่ามีไว้ทั้งสองทางน่าจะเป็นทางเลือกให้เพื่อนๆใช้ได้ง่ายกว่าครับ ตัวอย่าง 
https://react.semantic-ui.com/addons/responsive